src/main.js

```diff
@@ -1,9 +1,57 @@
 import React from 'react';
+import BreadCrumbs from './BreadCrumbs';
+import HomePage from './HomePage';
+import ResultPage from './ResultPage';
+import DetailResult from './DetailResult';
+import Data from './result.json'
 
 class Main extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      studentsData: Data.results,
+      displayPage: 'Home',
+      indStudent: {},
+    }
+    this.doRenderPageContent = this.doRenderPageContent.bind(this);
+    this.breadCrumbClick = this.breadCrumbClick.bind(this);
+    this.displayResult = this.displayResult.bind(this);
+  }
+
+  breadCrumbClick(displayPage) {
+    this.setState({
+      displayPage,
+    })
+  }
+
+  displayResult(displayPage, indStudent) {
+    this.setState({
+      displayPage,
```

Comment:
good use of enhanced object literals :+1:

Reply:
:)